scipy/special/basic.py

```diff
@@ -1446,6 +1446,15 @@ def ai_zeros(nt):
         Values of Ai(x) evaluated at first `nt` zeros of Ai'(x)
     aip : ndarray
         Values of Ai'(x) evaluated at first `nt` zeros of Ai(x)
+        
+    Examples
+    --------
+    >>> from scipy import special
```

Comment:
Here I think instead of just showing the output we can reorganize it slightly better
```python
>>> from scipy import special
>>> a, ap, ai, aip = special.ai_zeros(3)
>>> a
array([-2.33810741, -4.08794944, -5.52055983])
>>> ap
array([-1.01879297, -3.24819758, -4.82009921])
>>> ai
array([ 0.53565666, -0.41901548,  0.38040647])
>>> aip
array([ 0.70121082, -0.80311137,  0.86520403])
```

Reply:
Could you please add the same for the `bi_zeros` ?

Reply:
Sure.